packages/pwa/app/pwa-kit.config.schema.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Dumb question: is there a tool that helps us generate this schema? or we have do it manually everyt time we add a new config?

Reply:
There are lots of online tools that can help with this, example: https://jsonformatter.org/json-to-jsonschema